Drivers/gpio/client/gpio.c

```diff
@@ -70,34 +36,14 @@ void gpio_setDirection(const tGpioDef gpio, uint_fast8_t value )
     //memcpy(&sendbuf[4], tmpbuf, length);
     kySocket_send(&m_gpio_connection, sendbuf, length + sizeof(length));
 }
+void gpio_setPullUp(const tGpioDef gpio,uint_fast8_t value)
+{
+    char value_char = value ? '1' : '0';
```

Comment:
Where are the remaining parts of these functions? I can't see string constructed and no sending sections.